housekeeping: upgrade from system.reactive 3.x to system.reactive 4.x

ghuntley opened the pull request:
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**



**What is the current behavior? (You can also link to an open issue here)**



**What is the new behavior (if this is a feature change)?**



**Does this PR introduce a breaking change?**



**Please check if the PR fulfills these requirements**
- [ ] The commit follows our guidelines: https://github.com/reactiveui/reactiveui#contribute
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

olevett commented on src/ReactiveUI/ReactiveUI.csproj:
```diff
@@ -10,23 +10,23 @@
-  <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
-    <Compile Include="Platforms\net45\**\*.cs" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
+    <Compile Include="Platforms\net452\**\*.cs" />
```
I think for 4.x, this requires a few references (e.g. WindowsBase) - e.g. https://github.com/Reactive-Extensions/Rx.NET/blob/develop/Rx.NET/Source/src/System.Reactive/System.Reactive.csproj#L46

which I think explains the build failures.

ghuntley replied:
nah there's some wierdness with the system.reactive -preview000 package on NuGet. Spoken with Oren and I'm bumping us up to the nightly from MyGet. If this is 👍 from us then he will promote the System.Reactive build to NuGet.

ghuntley commented on .gitignore:
```diff
@@ -234,6 +234,7 @@ _Pvt_Extensions
 
 # Tools
 tools/
+tools/nuget/nuget.exe
```
We don't currently use a proper bootstrapper to download the initial `NuGet.exe` and instead rely on the existence of any version (which is updated to latest on each build). See https://github.com/reactiveui/ReactiveUI/blob/develop/build.cmd#L2

👍 to potentially moving to using the cake bootstrapper (proper) but downsides include if someone wants to build ReactiveUI and they are behind a corporate firewall then the bootstrapper will fail. I went with embedding the NuGet.exe binary into the repository because I'm often in such situations.

olevett replied:
Okay, makes sense - hadn't spotted we weren't using cakes bootstrapper :/

ghuntley commented:
Upgrading to Rx4x also fixes #1413. Previously installation would fail due to circular NuGet references.

# Android

```
PM> install-package reactiveui -pre -version 8.0.0-use-system-reac-0001

Attempting to gather dependency information for package 'reactiveui.8.0.0-use-system-reac-0001' with respect to project 'App3.Android', targeting 'MonoAndroid,Version=v7.1'
Gathering dependency information took 28.55 sec
Attempting to resolve dependencies for package 'reactiveui.8.0.0-use-system-reac-0001' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'reactiveui.8.0.0-use-system-reac-0001'
Resolved actions to install package 'reactiveui.8.0.0-use-system-reac-0001'
Retrieving package 'Microsoft.NETCore.Platforms 1.1.0' from 'nuget.org'.
Retrieving package 'Microsoft.Win32.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'NETStandard.Library 1.6.1' from 'nuget.org'.
Retrieving package 'reactiveui 8.0.0-use-system-reac-0001' from 'ReactiveUI development'.
Retrieving package 'Splat 2.0.0' from 'nuget.org'.
Retrieving package 'System.AppContext 4.3.0' from 'nuget.org'.
Retrieving package 'System.Collections 4.3.0' from 'nuget.org'.
Retrieving package 'System.Collections.Concurrent 4.3.0' from 'nuget.org'.
Retrieving package 'System.ComponentModel 4.3.0' from 'nuget.org'.
Retrieving package 'System.Console 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Contracts 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Debug 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Tools 4.3.0' from 'nuget.org'.
Retrieving package 'System.Diagnostics.Tracing 4.3.0' from 'nuget.org'.
Retrieving package 'System.Dynamic.Runtime 4.3.0' from 'nuget.org'.
Retrieving package 'System.Globalization 4.3.0' from 'nuget.org'.
Retrieving package 'System.Globalization.Calendars 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.Compression 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.Compression.ZipFile 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.FileSystem 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO.FileSystem.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq.Expressions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq.Queryable 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Http 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Sockets 4.3.0' from 'nuget.org'.
Retrieving package 'System.ObjectModel 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reactive 4.0.0-preview00001' from 'nuget.org'.
Retrieving package 'System.Reflection 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reflection.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Reflection.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Resources.ResourceManager 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Handles 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices.WindowsRuntime 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Numerics 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Algorithms 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Encoding 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.X509Certificates 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.Encoding 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.Encoding.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.RegularExpressions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Tasks 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Thread 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.ThreadPool 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Timer 4.3.0' from 'nuget.org'.
Retrieving package 'System.Xml.ReaderWriter 4.3.0' from 'nuget.org'.
Retrieving package 'System.Xml.XDocument 4.3.0' from 'nuget.org'.
Adding package 'Microsoft.NETCore.Platforms.1.1.0' to folder 'C:\tmp\App3\packages'
Added package 'Microsoft.NETCore.Platforms.1.1.0' to folder 'C:\tmp\App3\packages'
Added package 'Microsoft.NETCore.Platforms.1.1.0' to 'packages.config'
Successfully installed 'Microsoft.NETCore.Platforms 1.1.0' to App3.Android
Adding package 'Microsoft.Win32.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'Microsoft.Win32.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'Microsoft.Win32.Primitives.4.3.0' to 'packages.config'
Successfully installed 'Microsoft.Win32.Primitives 4.3.0' to App3.Android
Adding package 'Splat.2.0.0' to folder 'C:\tmp\App3\packages'
Added package 'Splat.2.0.0' to folder 'C:\tmp\App3\packages'
Added package 'Splat.2.0.0' to 'packages.config'
Successfully installed 'Splat 2.0.0' to App3.Android
Adding package 'System.AppContext.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.AppContext.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.AppContext.4.3.0' to 'packages.config'
Successfully installed 'System.AppContext 4.3.0' to App3.Android
Adding package 'System.Collections.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Collections.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Collections.4.3.0' to 'packages.config'
Successfully installed 'System.Collections 4.3.0' to App3.Android
Adding package 'System.Collections.Concurrent.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Collections.Concurrent.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Collections.Concurrent.4.3.0' to 'packages.config'
Successfully installed 'System.Collections.Concurrent 4.3.0' to App3.Android
Adding package 'System.ComponentModel.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.ComponentModel.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.ComponentModel.4.3.0' to 'packages.config'
Successfully installed 'System.ComponentModel 4.3.0' to App3.Android
Adding package 'System.Console.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Console.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Console.4.3.0' to 'packages.config'
Successfully installed 'System.Console 4.3.0' to App3.Android
Adding package 'System.Diagnostics.Contracts.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Contracts.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Contracts.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Contracts 4.3.0' to App3.Android
Adding package 'System.Diagnostics.Debug.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Debug.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Debug.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Debug 4.3.0' to App3.Android
Adding package 'System.Diagnostics.Tools.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tools.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tools.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tools 4.3.0' to App3.Android
Adding package 'System.Diagnostics.Tracing.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tracing.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tracing.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tracing 4.3.0' to App3.Android
Adding package 'System.Dynamic.Runtime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Dynamic.Runtime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Dynamic.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Dynamic.Runtime 4.3.0' to App3.Android
Adding package 'System.Globalization.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization 4.3.0' to App3.Android
Adding package 'System.Globalization.Calendars.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.Calendars.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.Calendars.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization.Calendars 4.3.0' to App3.Android
Adding package 'System.IO.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.4.3.0' to 'packages.config'
Successfully installed 'System.IO 4.3.0' to App3.Android
Adding package 'System.IO.Compression.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression 4.3.0' to App3.Android
Adding package 'System.IO.Compression.ZipFile.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.ZipFile.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.ZipFile.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression.ZipFile 4.3.0' to App3.Android
Adding package 'System.IO.FileSystem.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem 4.3.0' to App3.Android
Adding package 'System.IO.FileSystem.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem.Primitives 4.3.0' to App3.Android
Adding package 'System.Linq.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.4.3.0' to 'packages.config'
Successfully installed 'System.Linq 4.3.0' to App3.Android
Adding package 'System.Linq.Expressions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Expressions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Expressions.4.3.0' to 'packages.config'
Successfully installed 'System.Linq.Expressions 4.3.0' to App3.Android
Adding package 'System.Linq.Queryable.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Queryable.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Queryable.4.3.0' to 'packages.config'
Successfully installed 'System.Linq.Queryable 4.3.0' to App3.Android
Adding package 'System.Net.Http.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Http.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Http.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Http 4.3.0' to App3.Android
Adding package 'System.Net.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Primitives 4.3.0' to App3.Android
Adding package 'System.Net.Sockets.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Sockets.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Net.Sockets.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Sockets 4.3.0' to App3.Android
Adding package 'System.ObjectModel.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.ObjectModel.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.ObjectModel.4.3.0' to 'packages.config'
Successfully installed 'System.ObjectModel 4.3.0' to App3.Android
Adding package 'System.Reflection.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection 4.3.0' to App3.Android
Adding package 'System.Reflection.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Extensions 4.3.0' to App3.Android
Adding package 'System.Reflection.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Primitives 4.3.0' to App3.Android
Adding package 'System.Resources.ResourceManager.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Resources.ResourceManager.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Resources.ResourceManager.4.3.0' to 'packages.config'
Successfully installed 'System.Resources.ResourceManager 4.3.0' to App3.Android
Adding package 'System.Runtime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime 4.3.0' to App3.Android
Adding package 'System.Runtime.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Extensions 4.3.0' to App3.Android
Adding package 'System.Runtime.Handles.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Handles.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Handles.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Handles 4.3.0' to App3.Android
Adding package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices 4.3.0' to App3.Android
Adding package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' to App3.Android
Adding package 'System.Runtime.InteropServices.WindowsRuntime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.WindowsRuntime.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.WindowsRuntime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices.WindowsRuntime 4.3.0' to App3.Android
Adding package 'System.Runtime.Numerics.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Numerics.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Numerics.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Numerics 4.3.0' to App3.Android
Adding package 'System.Security.Cryptography.Algorithms.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Algorithms.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Algorithms.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Algorithms 4.3.0' to App3.Android
Adding package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Encoding 4.3.0' to App3.Android
Adding package 'System.Security.Cryptography.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Primitives.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Primitives 4.3.0' to App3.Android
Adding package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.X509Certificates 4.3.0' to App3.Android
Adding package 'System.Text.Encoding.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding 4.3.0' to App3.Android
Adding package 'System.Text.Encoding.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.Extensions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding.Extensions 4.3.0' to App3.Android
Adding package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.RegularExpressions 4.3.0' to App3.Android
Adding package 'System.Threading.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.4.3.0' to 'packages.config'
Successfully installed 'System.Threading 4.3.0' to App3.Android
Adding package 'System.Threading.Tasks.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Tasks.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Tasks.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Tasks 4.3.0' to App3.Android
Adding package 'System.Threading.Thread.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Thread.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Thread.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Thread 4.3.0' to App3.Android
Adding package 'System.Threading.ThreadPool.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.ThreadPool.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.ThreadPool.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.ThreadPool 4.3.0' to App3.Android
Adding package 'System.Threading.Timer.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Timer.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Timer.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Timer 4.3.0' to App3.Android
Adding package 'System.Xml.ReaderWriter.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Xml.ReaderWriter.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Xml.ReaderWriter.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.ReaderWriter 4.3.0' to App3.Android
Adding package 'System.Xml.XDocument.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Xml.XDocument.4.3.0' to folder 'C:\tmp\App3\packages'
Added package 'System.Xml.XDocument.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.XDocument 4.3.0' to App3.Android
Adding package 'NETStandard.Library.1.6.1', which only has dependencies, to project 'App3.Android'.
Adding package 'NETStandard.Library.1.6.1' to folder 'C:\tmp\App3\packages'
Added package 'NETStandard.Library.1.6.1' to folder 'C:\tmp\App3\packages'
Added package 'NETStandard.Library.1.6.1' to 'packages.config'
Successfully installed 'NETStandard.Library 1.6.1' to App3.Android
Adding package 'System.Reactive.4.0.0-preview00001' to folder 'C:\tmp\App3\packages'
Added package 'System.Reactive.4.0.0-preview00001' to folder 'C:\tmp\App3\packages'
Added package 'System.Reactive.4.0.0-preview00001' to 'packages.config'
Successfully installed 'System.Reactive 4.0.0-preview00001' to App3.Android
Adding package 'reactiveui.8.0.0-use-system-reac-0001' to folder 'C:\tmp\App3\packages'
Added package 'reactiveui.8.0.0-use-system-reac-0001' to folder 'C:\tmp\App3\packages'
Added package 'reactiveui.8.0.0-use-system-reac-0001' to 'packages.config'
Successfully installed 'reactiveui 8.0.0-use-system-reac-0001' to App3.Android
Executing nuget actions took 24.05 sec
Time Elapsed: 00:00:52.7121387
```

# iOS 

```
PM> install-package reactiveui -pre -version 8.0.0-use-system-reac-0001

Attempting to gather dependency information for package 'reactiveui.8.0.0-use-system-reac-0001' with respect to project 'App3.iOS', targeting 'Xamarin.iOS,Version=v1.0'
Gathering dependency information took 22.97 sec
Attempting to resolve dependencies for package 'reactiveui.8.0.0-use-system-reac-0001' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'reactiveui.8.0.0-use-system-reac-0001'
Resolved actions to install package 'reactiveui.8.0.0-use-system-reac-0001'
Found package 'Microsoft.NETCore.Platforms 1.1.0' in 'C:\tmp\App3\packages'.
Found package 'Microsoft.Win32.Primitives 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'NETStandard.Library 1.6.1' in 'C:\tmp\App3\packages'.
Found package 'reactiveui 8.0.0-use-system-reac-0001' in 'C:\tmp\App3\packages'.
Found package 'Splat 2.0.0' in 'C:\tmp\App3\packages'.
Found package 'System.AppContext 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Collections 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Collections.Concurrent 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.ComponentModel 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Console 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Diagnostics.Contracts 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Diagnostics.Debug 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Diagnostics.Tools 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Diagnostics.Tracing 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Dynamic.Runtime 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Globalization 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Globalization.Calendars 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.IO 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.IO.Compression 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.IO.Compression.ZipFile 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.IO.FileSystem 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.IO.FileSystem.Primitives 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Linq 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Linq.Expressions 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Linq.Queryable 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Net.Http 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Net.Primitives 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Net.Sockets 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.ObjectModel 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Reactive 4.0.0-preview00001' in 'C:\tmp\App3\packages'.
Found package 'System.Reflection 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Reflection.Extensions 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Reflection.Primitives 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Resources.ResourceManager 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.Extensions 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.Handles 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.InteropServices 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.InteropServices.WindowsRuntime 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Runtime.Numerics 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Security.Cryptography.Algorithms 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Security.Cryptography.Encoding 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Security.Cryptography.Primitives 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Security.Cryptography.X509Certificates 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Text.Encoding 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Text.Encoding.Extensions 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Text.RegularExpressions 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Threading 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Threading.Tasks 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Threading.Thread 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Threading.ThreadPool 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Threading.Timer 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Xml.ReaderWriter 4.3.0' in 'C:\tmp\App3\packages'.
Found package 'System.Xml.XDocument 4.3.0' in 'C:\tmp\App3\packages'.
Package 'Microsoft.NETCore.Platforms.1.1.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'Microsoft.NETCore.Platforms.1.1.0' to 'packages.config'
Successfully installed 'Microsoft.NETCore.Platforms 1.1.0' to App3.iOS
Package 'Microsoft.Win32.Primitives.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'Microsoft.Win32.Primitives.4.3.0' to 'packages.config'
Successfully installed 'Microsoft.Win32.Primitives 4.3.0' to App3.iOS
Package 'Splat.2.0.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'Splat.2.0.0' to 'packages.config'
Successfully installed 'Splat 2.0.0' to App3.iOS
Package 'System.AppContext.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.AppContext.4.3.0' to 'packages.config'
Successfully installed 'System.AppContext 4.3.0' to App3.iOS
Package 'System.Collections.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Collections.4.3.0' to 'packages.config'
Successfully installed 'System.Collections 4.3.0' to App3.iOS
Package 'System.Collections.Concurrent.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Collections.Concurrent.4.3.0' to 'packages.config'
Successfully installed 'System.Collections.Concurrent 4.3.0' to App3.iOS
Package 'System.ComponentModel.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.ComponentModel.4.3.0' to 'packages.config'
Successfully installed 'System.ComponentModel 4.3.0' to App3.iOS
Package 'System.Console.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Console.4.3.0' to 'packages.config'
Successfully installed 'System.Console 4.3.0' to App3.iOS
Package 'System.Diagnostics.Contracts.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Contracts.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Contracts 4.3.0' to App3.iOS
Package 'System.Diagnostics.Debug.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Debug.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Debug 4.3.0' to App3.iOS
Package 'System.Diagnostics.Tools.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tools.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tools 4.3.0' to App3.iOS
Package 'System.Diagnostics.Tracing.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Diagnostics.Tracing.4.3.0' to 'packages.config'
Successfully installed 'System.Diagnostics.Tracing 4.3.0' to App3.iOS
Package 'System.Dynamic.Runtime.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Dynamic.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Dynamic.Runtime 4.3.0' to App3.iOS
Package 'System.Globalization.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization 4.3.0' to App3.iOS
Package 'System.Globalization.Calendars.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Globalization.Calendars.4.3.0' to 'packages.config'
Successfully installed 'System.Globalization.Calendars 4.3.0' to App3.iOS
Package 'System.IO.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.IO.4.3.0' to 'packages.config'
Successfully installed 'System.IO 4.3.0' to App3.iOS
Package 'System.IO.Compression.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression 4.3.0' to App3.iOS
Package 'System.IO.Compression.ZipFile.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.IO.Compression.ZipFile.4.3.0' to 'packages.config'
Successfully installed 'System.IO.Compression.ZipFile 4.3.0' to App3.iOS
Package 'System.IO.FileSystem.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem 4.3.0' to App3.iOS
Package 'System.IO.FileSystem.Primitives.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.IO.FileSystem.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.IO.FileSystem.Primitives 4.3.0' to App3.iOS
Package 'System.Linq.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Linq.4.3.0' to 'packages.config'
Successfully installed 'System.Linq 4.3.0' to App3.iOS
Package 'System.Linq.Expressions.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Expressions.4.3.0' to 'packages.config'
Successfully installed 'System.Linq.Expressions 4.3.0' to App3.iOS
Package 'System.Linq.Queryable.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Linq.Queryable.4.3.0' to 'packages.config'
Successfully installed 'System.Linq.Queryable 4.3.0' to App3.iOS
Package 'System.Net.Http.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Net.Http.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Http 4.3.0' to App3.iOS
Package 'System.Net.Primitives.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Net.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Primitives 4.3.0' to App3.iOS
Package 'System.Net.Sockets.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Net.Sockets.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Sockets 4.3.0' to App3.iOS
Package 'System.ObjectModel.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.ObjectModel.4.3.0' to 'packages.config'
Successfully installed 'System.ObjectModel 4.3.0' to App3.iOS
Package 'System.Reflection.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection 4.3.0' to App3.iOS
Package 'System.Reflection.Extensions.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Extensions 4.3.0' to App3.iOS
Package 'System.Reflection.Primitives.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Reflection.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Reflection.Primitives 4.3.0' to App3.iOS
Package 'System.Resources.ResourceManager.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Resources.ResourceManager.4.3.0' to 'packages.config'
Successfully installed 'System.Resources.ResourceManager 4.3.0' to App3.iOS
Package 'System.Runtime.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime 4.3.0' to App3.iOS
Package 'System.Runtime.Extensions.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Extensions 4.3.0' to App3.iOS
Package 'System.Runtime.Handles.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Handles.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Handles 4.3.0' to App3.iOS
Package 'System.Runtime.InteropServices.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices 4.3.0' to App3.iOS
Package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.RuntimeInformation.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' to App3.iOS
Package 'System.Runtime.InteropServices.WindowsRuntime.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.InteropServices.WindowsRuntime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices.WindowsRuntime 4.3.0' to App3.iOS
Package 'System.Runtime.Numerics.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Runtime.Numerics.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Numerics 4.3.0' to App3.iOS
Package 'System.Security.Cryptography.Algorithms.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Algorithms.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Algorithms 4.3.0' to App3.iOS
Package 'System.Security.Cryptography.Encoding.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Encoding 4.3.0' to App3.iOS
Package 'System.Security.Cryptography.Primitives.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Primitives 4.3.0' to App3.iOS
Package 'System.Security.Cryptography.X509Certificates.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.X509Certificates 4.3.0' to App3.iOS
Package 'System.Text.Encoding.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding 4.3.0' to App3.iOS
Package 'System.Text.Encoding.Extensions.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Text.Encoding.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.Encoding.Extensions 4.3.0' to App3.iOS
Package 'System.Text.RegularExpressions.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.RegularExpressions 4.3.0' to App3.iOS
Package 'System.Threading.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Threading.4.3.0' to 'packages.config'
Successfully installed 'System.Threading 4.3.0' to App3.iOS
Package 'System.Threading.Tasks.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Tasks.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Tasks 4.3.0' to App3.iOS
Package 'System.Threading.Thread.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Thread.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Thread 4.3.0' to App3.iOS
Package 'System.Threading.ThreadPool.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Threading.ThreadPool.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.ThreadPool 4.3.0' to App3.iOS
Package 'System.Threading.Timer.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Threading.Timer.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Timer 4.3.0' to App3.iOS
Package 'System.Xml.ReaderWriter.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Xml.ReaderWriter.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.ReaderWriter 4.3.0' to App3.iOS
Package 'System.Xml.XDocument.4.3.0' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Xml.XDocument.4.3.0' to 'packages.config'
Successfully installed 'System.Xml.XDocument 4.3.0' to App3.iOS
Adding package 'NETStandard.Library.1.6.1', which only has dependencies, to project 'App3.iOS'.
Package 'NETStandard.Library.1.6.1' already exists in folder 'C:\tmp\App3\packages'
Added package 'NETStandard.Library.1.6.1' to 'packages.config'
Successfully installed 'NETStandard.Library 1.6.1' to App3.iOS
Package 'System.Reactive.4.0.0-preview00001' already exists in folder 'C:\tmp\App3\packages'
Added package 'System.Reactive.4.0.0-preview00001' to 'packages.config'
Successfully installed 'System.Reactive 4.0.0-preview00001' to App3.iOS
Package 'reactiveui.8.0.0-use-system-reac-0001' already exists in folder 'C:\tmp\App3\packages'
Added package 'reactiveui.8.0.0-use-system-reac-0001' to 'packages.config'
Successfully installed 'reactiveui 8.0.0-use-system-reac-0001' to App3.iOS
Executing nuget actions took 11.24 sec
Time Elapsed: 00:00:34.2417200
```

# UWP

```
PM> install-package reactiveui -pre -version 8.0.0-use-system-reac-0001
  GET https://api.nuget.org/v3/registration2-gz/reactiveui/index.json
  OK https://api.nuget.org/v3/registration2-gz/reactiveui/index.json 216ms
Restoring packages for C:\tmp\App3\App3\App3.UWP\project.json...
Restoring packages for C:\tmp\App3\App3\App3.UWP\project.json...
Installing reactiveui 8.0.0-use-system-reac-0001.
Committing restore...
Generating MSBuild file C:\tmp\App3\App3\App3.UWP\App3.UWP.nuget.targets.
Writing lock file to disk. Path: C:\tmp\App3\App3\App3.UWP\project.lock.json
Restore completed in 1.3 sec for C:\tmp\App3\App3\App3.UWP\App3.UWP.csproj.
Successfully installed 'reactiveui 8.0.0-use-system-reac-0001' to App3.UWP
Successfully installed 'Splat 2.0.0' to App3.UWP
Successfully installed 'System.Reactive 4.0.0-preview.2.build.18' to App3.UWP
Executing nuget actions took 813.37 ms
Time Elapsed: 00:00:04.1668265
```

ghuntley commented:
Closing this one out - no need to go up to v4.x yet - problem we were experiencing is related to NuGet.exe #1413
